lib/command/execute.rb

```diff
@@ -0,0 +1,15 @@
+module Command
```

Comment:
Meh - need to update my sample and remove that part - is should be much simpler and more explicit.